guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -1,17 +1,51 @@
 [id="glossary-of-terms-used-in-{project-context}_{context}"]
 = Glossary of terms used in {Project}
 
+{Project} is a complete lifecycle management tool for physical hosts, virtual machines, and cloud instances.
+Key features include automated host provisioning, configuration management, and content management including patch and errata management.
+You can automate tasks and quickly provision hosts, all through a single unified interface.
+
+This alphabetically ordered glossary provides an overview of {Project} related technical terms.
+
 [[Activation_key]]
-Activation key:: A token for host registration and subscription attachment.
-Activation keys define subscriptions, products, content views, and other parameters to be associated with a newly created host.
+Activation key::
+Activation keys are used by `subscription-manager` to register hosts to {Project}.
```

Comment:
```suggestion
Activation keys are used by xref:Subscription-manager[subscription manager] to register hosts to {Project}.
```
Not sure if we capitalize "subscription manager".

Reply:
Yes: https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#capitalization

Applied.